frontend/src/components/header/desktopMenu/styles.css

```diff
@@ -15,5 +15,6 @@
     margin-left: 3rem;
     font-size: 1rem;
     font-family: sans-serif;
+    font-weight: 300;
```

Comment:
300 of what? px, rem?

Reply:
![image](https://github.com/Hoocki/qpay/assets/75415127/7bacf6e4-f900-4733-aa1a-751216ab559f)